hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/volume/TestVolumeSet.java

```diff
@@ -138,7 +143,7 @@ public void testAddVolume() {
   public void testFailVolume() throws Exception {
 
     //Fail a volume
-    volumeSet.failVolume(volume1);
+    volumeSet.failVolume(volume1 + "/hdds");
```

Comment:
Why need add the "/hdds" now?

Reply:
Because the original test is using HddsVolume, originally, failVolume will use a function `getHddsRoot()` to append the "/hdds" for volume1, but now since the volume is StorageVolume, so I do it by hand.